### PR TITLE
APM-1675 Add _status flow for sandbox default proxy

### PR DESCRIPTION
**Pull request by Valentinolucas**
## Summary
* Add _status flow for sandbox default proxy 

## Reviews Required
* [x] Dev
* [ ] Test
* [ ] Tech Author
* [ ] Product Owner

## Review Checklist
:information_source: This section is to be filled in by the **reviewer**.

* [ ] I have reviewed the changes in this PR and they fill all or part of the acceptance criteria of the ticket, and the code is in a mergeable state.
* [ ] If there were infrastructure, operational, or build changes, I have made sure there is sufficient evidence that the changes will work.
* [ ] I have ensured the changelog has been updated by the submitter, if necessary.

### CODE REVIEW

File: proxies/sandbox/apiproxy/policies/KeyValueMapOperations.GetSecureVariables.xml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
```
Comment (alexhawdon) [inline proxies/sandbox/apiproxy/policies/KeyValueMapOperations.GetSecureVariables.xml:1]: Remove file

File: proxies/sandbox/apiproxy/proxies/default.xml [hunk 2]
```diff
@@ -19,8 +19,31 @@
         </Response>
         <Condition>(proxy.pathsuffix MatchesPath "/_ping") and ((request.verb = "GET") or (request.verb = "HEAD"))</Condition>
       </Flow>
+      <Flow name="StatusEndpoint">
+        <Description/>
+        <Request>
+          <Step>
```
Comment (alexhawdon) [inline proxies/sandbox/apiproxy/proxies/default.xml:25]: Remove step

**Review by alexhawdon — DISMISSED** [review]

File: proxies/sandbox/apiproxy/proxies/default.xml (hunk 2 above)
Comment (Valentinolucas) [reply proxies/sandbox/apiproxy/proxies/default.xml:25]: Done! Thanks!

File: proxies/sandbox/apiproxy/policies/KeyValueMapOperations.GetSecureVariables.xml (hunk 1 above)
Comment (Valentinolucas) [reply proxies/sandbox/apiproxy/policies/KeyValueMapOperations.GetSecureVariables.xml]: Removed Thanks!

**Review by Sourceless — APPROVED** [review]

**Review by Sourceless — APPROVED** [review]
LGTM 🚢

**Review by alexhawdon — APPROVED** [review]

**Pull request merged**